src/lgdo/types/lgdo.py

```diff
@@ -30,6 +34,41 @@ def form_datatype(self) -> str:
         """Return this LGDO's datatype attribute string."""
         pass
 
+    @abstractmethod
+    def view_as(
```

Comment:
I think we will want to have an argument for the number of rows to include in the view. This will be important for using this feature with the LH5Iterator. This class loads data into a fixed size LH5 table; on the last iteration, it does not fill all entries, so to correctly view the data we would only want to include the first n rows

Reply:
But since we return a view, wouldn't it be better if the slice happens on the user side?